src/System.Private.ServiceModel/tools/scripts/OpenFirewallPorts.cmd

```diff
@@ -0,0 +1,18 @@
+@echo off 
+setlocal
+
+REM 
+REM Start Open all ports WCF test services use
+REM
+
```

Comment:
Not blocking, but let's discuss the exe/script issue before investigating doing in a script.  The self-elevation and msbuild property parsing is a solved issue in the Bridge we can easily copy and paste. There is no need to reinvent the same code in script.